scripts/property_utils.sh

```diff
@@ -36,19 +30,24 @@ function set_property() {
   VALUE=$2
   FILE=$3
 
-  REGEXP="${1//\./\\\.}\s*="
-  if [ $(grep $REGEXP $FILE | wc -l) -eq 0 ]; then
-    write_prop $1 $2 $3
-  else
-    # delete line of specified property
+  REGEXP="${PROPERTY//\./\\.}\\s*="
+
+  if grep -q "$REGEXP" "$FILE"; then
+     # delete line of specified property
     LINE_COUNT=$(cnt_lines $FILE)
-    sed -i $SED_DISABLE_BACKUP "/${REGEXP}/d" $FILE
+
+    # Create new file and replace afterwards. Avoids corruption if update fails
+    sed -e /${REGEXP}/d "$FILE" > "${FILE}.new" && mv "${FILE}.new" "${FILE}"
```

Comment:
While I agree with other changes (thanks for the link in PR description btw, I learned a few stuff) I think this one is overkill.

set_property is intended to be used once or many times in a row in a script. This change implies as many copy and delete of files as there is properties to replace/overwrite. Seemless on SSD, maybe not so much on other systems.
Anyway, the work is half-done here since in case of failure, file "*.new" will be left on disk.

Unless this change is actually justified to make code more portable to Mac using `-e` option, is it ?